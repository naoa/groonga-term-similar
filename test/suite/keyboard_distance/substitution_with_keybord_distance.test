plugin_register commands/term_similar

table_create Tags TABLE_PAT_KEY ShortText UInt32

load --table Tags
[
{"_key": "Groonga"}
]

select Tags \
--output_columns 'keyboard_distance("Base", "Basd")' \
--limit 1 --command_version 2

select Tags \
--output_columns 'keyboard_distance("Base", "Basp")' \
--limit 1 --command_version 2

select Tags \
--output_columns 'keyboard_distance("Basz", "Bas=")' \
--limit 1 --command_version 2
