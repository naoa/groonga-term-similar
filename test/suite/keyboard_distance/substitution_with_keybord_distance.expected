plugin_register commands/term_similar
[[0,0.0,0.0],true]
table_create Tags TABLE_PAT_KEY ShortText UInt32
[[0,0.0,0.0],true]
load --table Tags
[
{"_key": "Groonga"}
]
[[0,0.0,0.0],1]
select Tags --output_columns 'keyboard_distance("Base", "Basd")' --limit 1 --command_version 2
[[0,0.0,0.0],[[[1],[["keyboard_distance","Object"]],[11.0]]]]
select Tags --output_columns 'keyboard_distance("Base", "Basp")' --limit 1 --command_version 2
[[0,0.0,0.0],[[[1],[["keyboard_distance","Object"]],[17.0]]]]
select Tags --output_columns 'keyboard_distance("Basz", "Bas=")' --limit 1 --command_version 2
[[0,0.0,0.0],[[[1],[["keyboard_distance","Object"]],[22.369316876853]]]]
